app/frontend/js/TestTransformation.js

```diff
@@ -0,0 +1,46 @@
+import {EditorState} from "@codemirror/state"
+import {EditorView, basicSetup} from "codemirror"
+import {json} from "@codemirror/lang-json"
+
+document.addEventListener('DOMContentLoaded', function () {
+
+
+  function sendData(form) {
```

Comment:
This is quite a big function, would it be easier to read if it was split up a bit?  It seems to do more than just send data

Reply:
Yeah this is a copy of another test feature that we have. I think we will revisit these when we have a javascript framework in the mix.

Reply:
Ahh, it could be that this is one of the code duplication things that was identified too then.

Reply:
Could be good to at least not have the duplication if it is mostly a copy paste.  Then it is easier to keep improving later